examples/counter/counter.js

```diff
@@ -7,7 +7,7 @@ const Counter = () => {
 
 	React.useEffect(() => {
 		const timer = setInterval(() => {
-			setCounter(prevCounter => prevCounter + 1);
+			setCounter(prevCounter => prevCounter + 1); // eslint-disable-line unicorn/prevent-abbreviations
```

Comment:
Perhaps it's better to either rename `prevCounter` to `previousCounter` to satisfy that linter rule or disable that rule for `examples` folder.

Reply:
> previousCounter

👍